android/build.gradle.kts

```diff
@@ -5,7 +5,7 @@ plugins {
     id("maven-publish")
     id("signing")
     id("org.jlleitschuh.gradle.ktlint")
-    kotlin("plugin.serialization") version "1.9.10"
+    kotlin("plugin.serialization") version "1.8.0"
```

Comment:
Previously we had this: https://github.com/open-feature/kotlin-sdk/commit/04717a192d4d54f2af9e289a5aaaca26da83798c

Reply:
we had this set to `1.8.10`